src/app/components/ATIAnalytics/canonical/sendBeaconOperaMiniScript/index.test.ts

```diff
@@ -24,6 +25,8 @@ describe('sendBeaconOperaMiniScript', () => {
     send: jest.fn(),
   };
 
+  const addStaticBeaconFn = eval(addSendStaticBeaconToWindow());
```

Comment:
Shall we rename so we don't need to do this:

`sendStaticBeacon: addStaticBeaconFn,`

```suggestion
  const sendStaticBeacon = eval(addSendStaticBeaconToWindow());
```